templates/travel/partials/results-navbar/results-navbar.snip.html

```diff
@@ -0,0 +1,112 @@
+{{!
+<!---
+Copyright 2016 The AMP Start Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+<!-- Results Navbar -->
+<header class="travel-results-navbar pt4 pr4 pl4">
+  <div class="px1 md-px2 flex justify-between items-stretch">
+    <div class="flex items-stretch">
+      <a href="travel.amp" class="h2 line-height-1 circle my1 md-my2">
+        {{> ../../../../img/travel/icon/logo.svg}}
+      </a>
+      <div class="ml3 flex items-center xs-hide sm-hide">
+        <amp-list
+          class="travel-block-list flex items-center"
+          layout="flex-item"
+          {{> ../util/amp-list-props.snip.html}}
+        >
+          <template type="amp-mustache">
+            {{=[[ ]]=}}
```

Comment:
what is this?

Reply:
This changes the the mustache template delimiters to `[[ ]]` so that the mustache template markup inside the `amp-mustache` isn't parsed by the gulp build. You can see the matching tag on L76 that switches back.

Reply:
https://stackoverflow.com/questions/11042926/how-does-one-use-a-literal-in-a-mustache-template

Reply:
Does this work ? I believe we used a custom escape on one other template (e-commerce) that we made work. Some of the mustache compilation is custom in ampstart . Let’s see if this works if it did I’ll swap out the stuff from other templates

Reply:
Can you link to the solution you are referencing?

Reply:
https://github.com/ampproject/ampstart/blob/master/templates/e-commerce/product-listing.amp.html#L63

Reply:
Our current solution works. Is it favorable to the custom-escape solution?